src/variables.tf

```
@@ -100,6 +100,12 @@ variable "s3_buckets_with_prefixes" {
   default     = {}
 }
 
+variable "s3_notification_events" {
+  type        = list(string)
+  description = "List of S3 events to trigger the Lambda notification"
+  default     = ["s3:ObjectCreated:*", "s3:ObjectRemoved:*"]
```

Please make the default list empty or the same as we have defaults in the module

```suggestion
  default     = []
```

---

Sure, no problem @goruha